Items/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -1,417 +1,418 @@
+DEX Requirement: 6
+
+	Can grapple limbs from range (DEX based grapple) and force movement 
+(1m for every 5 [DEX or STR]). Can make Disarm attempts at range. Grapple, 
+Shove, and Disarm attempts with the whip costs Actions as normal. This weapon
+has a Guard modifier of 0.
+
+== SPECO SP10 VAE Combat Knife $640 ==
+
+Damage: 45 + STR + 2*DEX + 1xBleed
+Size: Light
+Range:  Adjacent (1m)
+Accuracy Modifier: +1
+DEX Requirement: 4
+
+	Voltage Aligned Edge; ignores standard armor
+
+== SPECO SPC50 VAE Circular Saw $695 ==
+
+Damage: 55 + STR + 1xBleed
+Size: Two-handed
+Range: Adjacent (1m)
+Accuracy Modifier: +1
+STR Requirement: 4
+
+	Voltage Aligned Edge; ignores standard armor; can easily cut through most 
+materials. -50 on stealth rolls for being quiet during stealth attacks with 
+this weapon.
+
+== SPECO SP30 VAE Combat Sword $1245 ==
+
+Damage: 55 + 2*STR + 2*DEX + 1xBleed
+Size: One-handed
+Range: 2m
+Accuracy Modifier: +2
+DEX Requirement: 6
+
+	Voltage Aligned Edge; ignores standard armor
+
+===== LEVEL 10 REQUIREMENT WEAPONS =====
+
+== Eliazar E500 Assisted-Sledgehammer $811 ==
+
+Damage: 50 + 6*STR
+Size: Two-handed
+Range: 2m
+Accuracy Modifier: +0
+STR Requirement: 6
+
+	Considered concussive (explosive) damage. Hits armor points 
+before physical damage if the target's armor has defense against explosive (i.e. 
+concussive) damage. If armor roll is greater than 50%, target rolls a DC 50 
+fort save for a broken limb of your choice. -20 stealth rolls for being quiet 
+on stealth attacks. Reduces door-breaking DC by 30 per hit.
+
+== Eliazar E5500 Power Fist $880 ==
+
+Damage: 50 + 5*STR
+Size: Light
+Range: Adjacent (1m)
+Accuracy Modifier: +1
+STR Requirement: 5
+
+	Considered concussive (explosive) damage. Hits armor points before physical 
+damage if the target has defense against explosive (i.e. concussive) damage. If 
+armor roll is less than 50%, target rolls a DC 50% fort save for a being stunned 
+for 1 turn. -30% stealth rolls for being quiet on stealth attacks. Reduces 
+door-breaking DC by 45% per hit. This weapon has a Guard modifier of 0.
+
+== Muzashi VAE Katana $1354 ==
+
+Damage: 60 + STR + 3*DEX + 2xBleed
+Size: One-handed - Hybrid
+Range: 2m
+Accuracy Modifier: +2
+DEX Requirement: 6
+
+	Voltage Aligned Edge; ignores standard armor.
+
+== Eliazar Plasma Sword $987 ==
+
+Damage: 50 + STR + 2*DEX + 3xPlasma Burn
+Size: One-handed
+Range: 2m
+Accuracy Modifier: +2
+DEX Requirement: 4
+
+	Plasma effect reduces armor by 1 level as usual, but this effect applies 
+during the sword hit (instead of after). If the target has a standard 
+(non-Nanite) shield, all plasma burn is canceled, the shield takes 30 damage and 
+the plasma sword passes through the shield.
+
+
+==============================
+THROWN WEAPONS
+==============================
+
+	These weapons work similarly to guns. However your range is determined by 
+your DEX or STR and your miss chance is reduced by your [Combat PER Mod]. Like 
```

Comment:
Sentence structure: since miss chance with guns is also reduced by combat Per mod, "your miss chance is reduced by your [Combat PER Mod]" belongs in the previous sentence (or any sentence except for one qualified by "however")